rasa_nlu/featurizers/regex_featurizer.py

```diff
@@ -35,15 +37,19 @@ class RegexFeaturizer(Featurizer):
 
     requires = ["tokens"]
 
-    def __init__(self, component_config=None, known_patterns=None):
+    def __init__(self, component_config=None,
+                 known_patterns=None, lookup_tables=[]):
```

Comment:
this needs to be `=None` and within `__init__` you can do `lookup_tables = lookup_tables  or []`, we can't have `[]` as a default argument (as it is mutable)

Reply:
ok I changed this